yo/app/scripts/directives/data-config-csv.js

```diff
@@ -93,31 +96,27 @@ angular.module('dmpApp')
         function extractFromResource(dataResource) {
             if (dataResource.configurations) {
 
-                var latestConfig = loDash.max(dataResource.configurations, 'id');
+                var latestConfig = dataResource.configurations[0];
```

Comment:
This doesn't seem too correct, we probably have to address this at some point. IIRC you've already mentioned this at the last meeting. Can we have a TODO or an issue?